include/exadg/solvers_and_preconditioners/multigrid/multigrid_input_parameters.h

```diff
@@ -164,20 +171,51 @@ struct AMGData
 {
   AMGData()
   {
-    data.smoother_sweeps = 1;
-    data.n_cycles        = 1;
-    data.smoother_type   = "ILU";
+    amg_type = AMGType::Trilinos;
+
+    trilinos_data.smoother_sweeps = 1;
+    trilinos_data.n_cycles        = 1;
+    trilinos_data.smoother_type   = "ILU";
+
+    boomer_data.n_sweeps_coarse = 1;
+    boomer_data.max_iter        = 1;
+    boomer_data.relaxation_type_down =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
+    boomer_data.relaxation_type_up =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
+    boomer_data.relaxation_type_coarse =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
   };
 
   void
   print(ConditionalOStream & pcout)
   {
-    print_parameter(pcout, "    Smoother sweeps", data.smoother_sweeps);
-    print_parameter(pcout, "    Number of cycles", data.n_cycles);
-    print_parameter(pcout, "    Smoother type", data.smoother_type);
+    print_parameter(pcout, "    AMG type", enum_to_string(amg_type));
+
+    if(amg_type == AMGType::Trilinos)
+    {
+      print_parameter(pcout, "    Smoother sweeps", trilinos_data.smoother_sweeps);
+      print_parameter(pcout, "    Number of cycles", trilinos_data.n_cycles);
+      print_parameter(pcout, "    Smoother type", trilinos_data.smoother_type);
+    }
+    else if(amg_type == AMGType::Boomer)
+    {
+      print_parameter(pcout, "    Smoother sweeps", boomer_data.n_sweeps_coarse);
+      print_parameter(pcout, "    Number of cycles", boomer_data.max_iter);
+      // TODO: add enum_to_string function (in dealii?)
+      print_parameter(pcout, "    Smoother type down", (int)boomer_data.relaxation_type_down);
+      print_parameter(pcout, "    Smoother type up", (int)boomer_data.relaxation_type_up);
+      print_parameter(pcout, "    Smoother type coarse", (int)boomer_data.relaxation_type_coarse);
+    }
+    else
+    {
+      AssertThrow(false, ExcNotImplemented());
+    }
   }
 
-  TrilinosWrappers::PreconditionAMG::AdditionalData data;
+  AMGType                                              amg_type;
+  TrilinosWrappers::PreconditionAMG::AdditionalData    trilinos_data;
```

Comment:
ml_data for consistency?

Reply:
Done in commit [d0a2ff9](https://github.com/exadg/exadg/pull/58/commits/d0a2ff93d87cd124a4426a80462886196481f72a).